packages/mui-base/src/SelectUnstyled/useSelect.ts

```diff
@@ -72,7 +72,7 @@ function useSelect<TValue>(props: UseSelectParameters<TValue>) {
     }
   }, [listboxFocusRequested]);
 
-  const updateListboxRef = (listboxElement: HTMLUListElement) => {
+  const updateListboxRef = (listboxElement: HTMLUListElement | null) => {
```

Comment:
This was unsound

Reply:
We would basically need to always add `| null` when using ref callbacks no?